admin_ui/src/components/TableNav.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-    <div>
+    <div class="sidebar_wrapper">
```

Comment:
What's this for?

Reply:
This is for a menu sidebar of equal width between the `Home` page and other pages. Now this is not the case because the sidebar menu on the `Home` page is wider than on other pages. I can remove it if you want?